src/utils/shell.ts

```diff
@@ -37,12 +37,12 @@ export function openUrlInExternalBrowser(url?: string) {
 }
 
 export function openDocumentation() {
-  trackEvent('WINDOW_HELP_LINK', {linkID: 'documentation', osType: os.type});
+  trackEvent(WINDOW_HELP_LINK, {linkID: 'documentation', osType: os.type});
```

Comment:
do we really need to add os.type to this event? don't we get it automatically through nucleus?

Reply:
yeah we have - There is no need to keep this value You are right